docs/rst/user_manual/configuration.rst

```diff
@@ -230,224 +195,149 @@ The following is the set of Quality of Services that are configurable for a topi
+-----------------
 
-    builtin-topics:
-      - name: rt/chatter
-        type: std_msgs::msg::dds_::String_
+``specs`` supports a ``threads`` **optional** value that allows the user to set a maximum number of threads
+for the internal :code:`ThreadPool`.
+This ThreadPool allows to limit the number of threads spawned by the application.
+This improves the performance of the data transmission between Participants.
 
-    blocklist:
-      - name: "rq/*"
-      - name: "rr/*"
+This value should be set by each user depending on each system's characteristics.
+In case this value is not set, the default number of threads used is :code:`12`.
 
+.. _user_manual_configuration_remove_unused_entities:
 
-Allowlist and blocklist collision
-"""""""""""""""""""""""""""""""""
+Remove Unused Entities
+----------------------
 
-In the following example, the ``HelloWorldTopic`` topic is both in the ``allowlist`` and (implicitly) in the
-``blocklist``, so according to the ``blocklist`` preference rule this topic is blocked.
-Moreover, only the topics present in the allowlist are relayed, regardless of whether more topics are dynamically
-discovered in the DDS network.
-In this case the forwarded topics are ``AllowedTopic1`` with data type ``Allowed``
-and ``AllowedTopic2`` regardless of its data type.
+``specs`` supports a ``remove-unused-entities`` **optional** value that configures the deletion of unused internal entities in the |ddsrouter|.
+By default, unused internal entities are *not* removed.
+Thus, when the |ddsrouter| discovers a Subscriber, the |ddsrouter| creates entities in all of its participants, and these entities stay up even after the Subscriber disconnects.
 
-.. code-block:: yaml
+At times it can be useful to remove the internal entities that are not being used.
+Consider the following example.
+Two *DDS Routers* are communicating through a WAN connection, when the last of the external Subscribers to which they are forwarding data disconnects.
+By default, the internal entities of the *DDS Routers* would *not* be removed, so the *DDS Routers* would keep consuming bandwidth, even though the data is never read.
+By setting the ``remove-unused-entities`` option to ``true``, the internal entities of the |ddsrouter| would be removed, and the *DDS Routers* would stop communicating and free up the bandwidth.
 
-    allowlist:
-      - name: AllowedTopic1
-        type: Allowed
-      - name: AllowedTopic2
-        type: "*"
-      - name: HelloWorldTopic
-        type: HelloWorld
+.. warning::
+  At the time being, the removal of unused entities is incompatible with the `Transient-Local Durability QoS <https://fast-dds.docs.eprosima.com/en/latest/fastdds/dds_layer/core/policy/standardQosPolicies.html#durabilityqospolicy>`_.
 
-    blocklist:
-      - name: "*"
-        type: HelloWorld
+QoS
+---
 
+``specs`` supports a ``qos`` **optional** tag to configure the default values of the :ref:`Topic QoS <user_manual_configuration_topic_qos>`.
```

Comment:
I would add here two notes regarding:

-  `builtin-topics` case, reminding that in such case they get default values, which would be overwritten by `specs` qos. 
-  non `builtin-topics` case, where `specs` qos would only have effect on non-dds qos, since default qos are overwritten by qos obtained via topic's discovery.